gef-ui/templates/partials/upload-supporting-documents.njk

```diff
@@ -108,5 +109,5 @@
 {% endblock %}
 
 {% block scripts %}
-  <script src="/assets/js/fileUpload.js"></script>
+  <script id="multi-file-upload" src="/assets/js/multiFileUpload.js" uploadCsrf="{{ uploadCsrf }}" ></script>
```

Comment:
I'm not quite sure what you mean sorry.

The attribute is so that `uploadCsrf` can be accessed from the multiFileUpload script. Is there a better way of passing it in from here?

Reply:
Have updated on a peer call.